src/utils/workflow-utils.js

```diff
@@ -45,9 +45,10 @@ const isValidQuestionWorkflow = (workflow) => {
 
   const questionNotTooLong = firstTask.question.length < 200
   const notTooManyShortcuts = shortcut ? shortcut.answers.length <= 2 : true
-  const doesNotUseFeedback = firstTask.feedback ? !firstTask.feedback.enabled : true;
+  const usesFeedback = firstTask.feedback ? firstTask.feedback.enabled : false;
 
-  if (hasSingleTask && questionNotTooLong && notTooManyShortcuts && doesNotUseFeedback) {
+  // Feedback will have multiple tasks so you chec that the workflow has a single task OR is feedback.
+  if ((hasSingleTask || usesFeedback)&& questionNotTooLong && notTooManyShortcuts) {
```

Comment:
I think the `doesNotUseFeedback`/`usesFeedback` check can be eliminated from `isValidQuestionWorkflow()` because a question task workflow can either have feedback or not and be valid.

The current comment statement is not correct: feedback does not necessitate multiple tasks. Therefore, requiring the `(hasSingleTask || usesFeedback)` logic statement is not necessary.

Reply:
@coryzoon Pinging you here as the PR is getting ready to be closed out.